### PR TITLE
Translate test

### PR DESCRIPTION
**Pull request by tanoueakira**
Translate noTone.adoc

### CODE REVIEW

**Comment by akash73** [comment]
HI, we have added an handy guide to explain better how to translate:
in your PR please do not translate first section : 

 ---
 -title: noTone()		 +タイトル: noTone()
  title_expanded: ""		  title_expanded: ""
 -categories: [ "Functions" ]		 +カテゴリ: [ "Functions" ]
 -subCategories: [ "Advanced I/O" ]		 +サブカテゴリ: [ "Advanced I/O" ]
  ---

here is the guide : 
https://create.arduino.cc/projecthub/Arduino_Genuino/contribute-to-the-arduino-reference-af7c37

basically you don't need to fork, just edit the page from the github interface, once done, you can create a PR .

thanks!

**Comment by tanoueakira** [comment]
Leave a comment

**Comment by tanoueakira** [comment]
Thank you for your commment!
